connector/saml/saml.go

```diff
@@ -598,3 +647,34 @@ func before(now, notBefore time.Time) bool {
 func after(now, notOnOrAfter time.Time) bool {
 	return now.After(notOnOrAfter.Add(allowedClockDrift))
 }
+
+// deflate compresses the data
+func deflate(data []byte) ([]byte, error) {
```

Comment:
nit: just call this "compressRequest" and "decompressRequest"

Reply:
Yes